app/models/manageiq/providers/network_manager.rb

```diff
@@ -96,5 +96,21 @@ def self.find_object_for_belongs_to_filter(name)
       name.gsub!(" #{self::PROVIDER_NAME}", "")
       includes(:parent_manager).find_by(:parent_managers_ext_management_systems => {:name => name})
     end
+
+    def create_cloud_subnet_queue(userid, options = {})
+      task_opts = {
+        :action => "creating Cloud Subnet for user #{userid}",
+        :userid => userid
+      }
+      queue_opts = {
+        :class_name  => self.class.name,
+        :method_name => 'create_cloud_subnet',
+        :instance_id => id,
+        :role        => 'ems_operations',
+        :zone        => ext_management_system.my_zone,
```

Comment:
Since we're in the ext_management_system (network_manager inherits from ext_management_system) you can just use `my_zone` here like https://github.com/ManageIQ/manageiq-providers-openstack/pull/721/files#diff-be9d1445a5a061184cb3914ea8bdec3ca1a42e7682144ee4f67a2eee0fb5f1a5L131